doc/modules/ROOT/pages/cops_bundler.adoc

```diff
@@ -46,7 +46,7 @@ gem 'rubocop', groups: [:development, :test]
 | Name | Default value | Configurable values
 
 | Include
-| `+**/*.gemfile+`, `+**/Gemfile+`, `+**/gems.rb+`
```

Comment:
Looking at the preview (https://github.com/rubocop-hq/rubocop/blob/f5148812b2843e4dec3efac73e0dd870dbef4d50/doc/modules/ROOT/pages/cops_bundler.adoc)
it seems we needed those `+` signs as well, otherwise boldness gets applied because of the `*`.

Reply:
I updated the rake task. Bold style has been removed.
https://github.com/koic/rubocop/blob/generate_cops_documentation_for_ascii_doc/manual/cops_bundler.md#configurable-attributes